### ISSUE TITLE
TorchSystem.js

### ISSUE THREAD

**Issue by Oops86** [kolton/d2bot-with-kolbot#488]
hey, 
I have a problem with dropping keys by farmers - the basic function does not include recent fix that to drop an item you cannot have opened stash. Please help to fix that.

**Comment by noah-** [comment]
Yes, this should be a simple change. You need to move the item(s) to your inventory first. Maybe someone can do a quick pull request for this?

**Comment by Oops86** [comment]
it does not work like this, bot goes to stash and nothing happens

**Comment by ross-weir** [comment]
@Oops86 

Are you using the most up-to-date versions of other files? What exactly happens? Does it just walk to stash and then not open the stash? It works for me on bnet. 👍 

Why do we have to transfer keys to inventory when we can just open stash, item to cursor, close stash, drop item like it does during auto muling? Or is auto muling currently broken as well?

**Comment by Oops86** [comment]
Yes it dosnt open stash at all - you removed this condition in TorchSystem.js - line 157, im confused :P, do i miss sth else? thanks

**Comment by ross-weir** [comment]
It's not needed because `drop()` will open the stash if the item that's getting dropped is in the stash.

If it's not opening the stash you probably don't have the most up to date files. Make sure you have the latest [Prototypes.js](https://github.com/kolton/d2bot-with-kolbot/blob/master/d2bs/kolbot/libs/common/Prototypes.js)

**Comment by Oops86** [comment]
Thank you :)
